tests/failover.go

```diff
@@ -449,42 +449,72 @@ func (oa *operatorActions) CheckOneEtcdDownOrDie(operatorConfig *OperatorConfig,
 		if err != nil {
 			return err
 		}
-		glog.V(4).Infof("all clusters is available")
+		glog.V(4).Infof("all clusters are available")
+		return nil
```

Comment:
all should be available within 10 minutes.  should not return here.

Reply:
it's ok